tests/test_suite.py

```diff
@@ -216,10 +216,32 @@ def auto_select_xccdf_id(datastream, bench_number):
     return xccdf_ids[bench_number]
 
 
+def get_datastreams():
+    ds_glob = "ssg-*-ds.xml"
+    build_dir_path = [os.path.dirname(__file__), "..", "build"]
+    datastreams = glob(os.path.sep.join(build_dir_path + [ds_glob]))
+    return datastreams
+
+
+def get_unique_datastream():
+    datastreams = get_datastreams()
+    if len(datastreams) == 1:
+        return datastreams[0]
+    msg = ("Unable to auto-detect the datastream, "
+           "expected to find a single one in the build dir, but")
```

Comment:
I wouldn't say that it is expected that there is only single datastream in build dir, I have othen more than one.